Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -49,10 +49,21 @@ - (instancetype)initWithURL:(NSURL *)URL
 - (void)setUp
 {
   if (!_url) {
+#define AUTO_IP
+// Uncomment the following line to disable the AUTO_IP configuration of the debugger
+// websocket.  See also the AUTO_IP logic in AppDelegate.m
+//#undef AUTO_IP
+#ifdef AUTO_IP
+      NSString *serverIP = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"SERVER_IP"];
+      NSInteger port = [standardDefaults integerForKey:@"websocket-executor-port"] ?: 8081;
+      NSString *debugUrlString = [NSString stringWithFormat:@"http://%@:%zd/debugger-proxy", serverIP, port];
```

Comment:
It runs for me.  However, I am using the 8.x simulator.  You may be right about it not working in the 9.x simulators.

I'll look at the fix for this.